tests/GridFS/BucketFunctionalTest.php

```diff
@@ -814,6 +814,10 @@ public function testUploadingFirstFileCreatesIndexes(): void
 
     public function testExistingIndexIsReused(): void
     {
+        // The collections may exist from other tests, ensure they are removed before and after the test
+        $this->dropCollection($this->getDatabaseName(), 'fs.chunks');
+        $this->dropCollection($this->getDatabaseName(), 'fs.files');
```

Comment:
By calling `dropCollection`, the collection is also removed on `tearDown`.

Reply:
My suggestion to explain the optimization in a comment within `tearDown()` would pair nicely with this comment.